frontend/src/App.tsx

```diff
@@ -79,6 +84,31 @@ const App = (): React.ReactElement => {
                 path={Routes.HOOKS_PAGE}
                 component={HooksDemo}
               />
+              <PrivateRoute
+                exact
+                path={Routes.ADMIN_DASHBOARD_PAGE}
+                component={AdminDasboard}
+              />
+              <PrivateRoute
+                exact
+                path={Routes.COURSE_OVERVIEW_DASHBOARD_BASE_ROUTE}
+                component={CourseDashboard}
+              />
+              <PrivateRoute
+                exact
+                path={`${Routes.COURSE_OVERVIEW_DASHBOARD_BASE_ROUTE}/:id`}
+                component={CourseOverview}
+              />
+              <PrivateRoute
+                exact
+                path={Routes.ADMIN_COURSE_EDITOR_BASE_ROUTE}
+                component={CourseEditorOverview}
+              />
```

Comment:
This raises an interesting question, but I don't think we should have pages for the base routes. For example, I don't think `/admin/course` should be valid, but I think `/admin/course/<id>` should be. Same goes for the course base route. What are peoples thoughts on this? I'm simply not sure how we would use them.

Reply:
These base routes are for the overview pages, for example /admin/course would show a list of courses. They're in the mockup as the "Manage Courses" tab, but I'm not sure if I misinterpreted it.